subcommands/http/http.go

```diff
@@ -64,6 +82,18 @@ func doPost(cmd *cobra.Command, args []string) {
 	printResponse(api.RawPost(args[0], readData(cmd), readHeaders(args[1:])))
 }
 
+func doPut(cmd *cobra.Command, args []string) {
+	printResponse(api.RawPut(args[0], readData(cmd), readHeaders(args[1:])))
```

Comment:
Maybe just "doIt(<method name>, ...)" instead of N Raw<method> functions?

Reply:
That was what I wanted to do initially.
But, in due to small differences between `api.RawXXX` functions, it would require more coding.

I opted to go for the least amount of effort approach.